### PR TITLE
fix: Fixed Suggest a presentation UI

### LINKED ISSUE #136: Update "Projects" > "Suggest a presentation" page UI
### What feature do you want to see added?

Currently the spacing of the navigation bar on the left hand side of the UI of the "Projects" > "Suggest a presentation" page seems inconsistent with the rest of the versioned docs. Also, the contact form on the right hand side needs a bit of dressing up:

Screenshot 2024-10-13 at 12 21 52 AM

We would like to update the UI of this page to make it more visually pleasing and consistent with the rest. 

### Upstream changes

N/A

### PR DESCRIPTION
**Pull request by pateldivyesh1323**
Fixes #136 

Fixed: 

Location: "Projects" > "Suggest a presentation"

1. Spacing of navigation bar of left side
2. Brushed up UI of contact form

### Images

- Before 

image

image

- After

image

image

### CODE REVIEW

File: docs/projects/modules/jam/pages/suggest.adoc [hunk 1]
```diff
@@ -1,45 +1,118 @@
-= Suggest a presentation
+= Suggest a presentation
```
Comment (krisstern) [inline docs/projects/modules/jam/pages/suggest.adoc:1]:
```suggestion
= Suggest a presentation
```

**Review by krisstern — CHANGES_REQUESTED** [review]

File: docs/projects/modules/jam/pages/suggest.adoc (hunk 1 above)
Comment (krisstern) [reply docs/projects/modules/jam/pages/suggest.adoc:1]: Thanks @pateldivyesh1323 for the PR! Everything else looks good except for one minor change requested.

**Comment by pateldivyesh1323** [comment]
@krisstern Done

**Review by krisstern — APPROVED** [review]
Hey @pateldivyesh1323 everything looks good! Thanks for the PR

**Pull request merged**